### ISSUE TITLE
WaterHeater:HeatPump:PumpedCondenser part-load ratio calculation failed warning

### ISSUE THREAD

**Issue by mjwitte** [NREL/EnergyPlus#5400]
Helpdesk ticket 11029
User file generates these warnings:

```
   ** Warning ** WaterHeater:HeatPump:PumpedCondenser "HPWATERHEATER_DHW"
   **   ~~~   ** Heat pump water heater compressor part-load ratio calculation failed: PLR limits of 0 to 1 exceeded. Part-load ratio used = 0.813
   **   ~~~   ** Please send this information to the EnergyPlus support group.
   **   ~~~   ** This error occured in float mode. Environment=RUN PERIOD 1, at Simulation time=05/25 05:00 - 05:05

   *************  ** Warning ** WaterHeater:HeatPump:PumpedCondenser "HPWATERHEATER_DHW": Part-load ratio calculation failed in float mode warning continues. Part-load ratio statistics follow.
   *************  **   ~~~   **   This error occurred 32 total times;
   *************  **   ~~~   **   during Warmup 0 times;
   *************  **   ~~~   **   during Sizing 0 times.
   *************  **   ~~~   **   Max=0.920741  Min=0.159959
```

Run with Los Angeles, CA weather.

**Comment by rraustad** [comment]
Interesting. The defect file uses EMS to adjust the tank SP. This in turn confuses the model to believe the HP increased the water temperature and needed to find a PLR below 1.

```
EnergyManagementSystem:Program,
  TankControl,             !- Name
  IF (CurrentTime>4.7) && (CurrentTime<4.8),  !- Program Line 1
  SET LRdminTemp= LRoomTemp,  !- Program Line 2
  ELSE,                    !- A4
  SET LRdminTemp=@MIN  LRoomTemp  LRdminTemp,  !- A5
  ENDIF,                   !- A6
  IF LRdminTemp<20.5,      !- A7
  SET AWHPSPA=45,          !- A8
  ELSE,                    !- A9
  SET AWHPSPA=12,          !- A10
  ENDIF,                   !- A11
  SET HeatingTankSPA= AWHPSPA-16;
```

Looks like missing logic for HPWH.

**Comment by rraustad** [comment]
See PR #5761

**Comment by Myoldmopar** [comment]
Closed via #5761
